app/controllers/legacy/sessions_controller.rb

```diff
@@ -301,5 +319,11 @@ def store_authorization_url_as_fallback
 
       store_fallback(url: authorization_url) unless authorization_url.nil?
     end
+
+    def field_error!(on:, code:, message:)
+      @errors ||= Lev::Errors.new
```

Comment:
Don't know. That is the legacy flow though, so it really doesn't need to be modified — and ideally it shouldn't be modified because it's battle tested and has been working for years, so to prevent errors in case that we ever switch the feature flags off.